dev/Dockerfile.anvil

```diff
@@ -28,3 +28,13 @@ COPY . .
 
 RUN chmod +x ./dev/gen-anvil-state \
     && ./dev/gen-anvil-state
+
+# Cleanup the builder image.
+# Instead of using a builder image, remove the build directory, \
+# and copy the files to the app directory.
+# This way it's not needed to run duplicate steps (apt-get, foundryup, etc)
+WORKDIR /app
+
+RUN cp /build/deployments/anvil_localnet/anvil-state.json . \
+    && cp /build/deployments/anvil_localnet/anvil-state-info.json . \
+    && rm -rf /build
```

Comment:
can you also set the ENTRYPOINT to anvil + the state load? It can always be overridden but it makes using it easier.

Reply:
Done. 👍🏻